src/hotspot/share/c1/c1_Runtime1.cpp

```diff
@@ -644,14 +644,11 @@ address Runtime1::exception_handler_for_pc(JavaThread* thread) {
   oop exception = thread->exception_oop();
   address pc = thread->exception_pc();
   // Still in Java mode
-  DEBUG_ONLY(ResetNoHandleMark rnhm);
```

Comment:
These ResetNoHandleMarks have been there since before mercurial.  I don't see any reason for the ResetNoHandleMark here historically either, since there isn't a NoHandleMark outside this call.